packages/service-library/src/servicelib/archiving_utils.py

```diff
@@ -284,4 +339,8 @@ def prune(self, exclude: Set[Path]) -> None:
                 p.rmdir()
 
 
-__all__ = ["archive_dir", "unarchive_dir", "PrunableFolder"]
+__all__ = [
```

Comment:
MINOR: tuple to suggest that this variable is inmutable

Reply:
done